presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -262,26 +269,19 @@ public Node visitCreateTable(SqlBaseParser.CreateTableContext context)
         if (context.COMMENT() != null) {
             comment = Optional.of(((StringLiteral) visit(context.string())).getValue());
         }
+        List<Property> properties = ImmutableList.of();
+        if (context.properties() != null) {
+            properties = visit(context.properties().property(), Property.class);
+        }
         return new CreateTable(
                 getLocation(context),
                 getQualifiedName(context.qualifiedName()),
                 visit(context.tableElement(), TableElement.class),
                 context.EXISTS() != null,
-                processProperties(context.properties()),
+                properties,
                 comment);
     }
 
-    private Map<String, Expression> processProperties(PropertiesContext propertiesContext)
```

Comment:
We should be able to keep this method and have it return `List<Property>`. This avoids duplicating the code. You might be able to use the existing `visit()` helper.

Reply:
There is not much duplicating logic. If you're referring to `if (context.properties() != null)`, this is not extracted elsewhere in the file.